src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,52 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
```

Comment:
it would be better not to import it again, but only for the first time and immediately merge